### PR TITLE
Ethereum whitepaper improvements [Fixes #5773]

### LINKED ISSUE #5773: Make Ethereum whitepaper more consistent and enhance semantic
**Describe the Enhancement**

When I read the whitepaper, several things have caught my eye:
- in the `A Next-Generation Smart Contract and Decentralized Application Platform` some links have quotation marks included and some have not
- the link to the pdf doesn't really emphasizes that it leads to the original whitepaper
- the header `Long-Term Supply Growth Rate (percent)` is just bold text and not really a header, which is not very semantic
- some cursive sections are missing a period a the end. all other cursive sections have a period
- the link to the history of the whitepaper is bare, which doesn't look good and also differs from how the other links are implemented

### PR DESCRIPTION
**Pull request by noahliechti**
## Description
- remove all quotation marks from links (enhance consistency)
- emphasize that the link to the pdf leads to the original whitepaper
- make the header `Long-Term Supply Growth Rate (percent)` more semantic by using `####` instead of making it just bold 
- add a period to all cursive sections (enhance consistency)
- use a text for the bare link to the history of the whitepaper (looks better, enhance consistency) 

## Related Issue
fixes #5773

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
LGTM.. @minimalsm Given we're linking out to the PDF, I assume we're cool with small adjustments like this

File: src/content/whitepaper/index.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ _This introductory paper was originally published in 2014 by Vitalik Buterin, th
 
 _While several years old, we maintain this paper because it continues to serve as a useful reference and an accurate representation of Ethereum and its vision. To learn about the latest developments of Ethereum, and how changes to the protocol are made, we recommend [this guide](/learn/)._
 
-[Open the Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
+[Open the original Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
```
Comment (minimalsm) [inline src/content/whitepaper/index.md:15]:
```suggestion
[Open the Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
```

I don't agree with this change. What we had proposed in the original issue about reverting the whitepaper was meant to create one canonical whitepaper, every other version we don't consider to be the valid 'true' whitepaper. Also, this implies that the whitepaper that follows is somehow different, which it isn't (or at least shouldn't be)

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks @noahliechti 

Agree with most changes, suggested one change to the PDF link.

**Review by corwintines — APPROVED** [review]
Merged @minimalsm suggestion, looks good to me. Thanks, @noahliechti!

File: src/content/whitepaper/index.md (hunk 1 above)
Comment (noahliechti) [reply src/content/whitepaper/index.md]: Makes sense to me. Thanks for the explanation.

**Review by minimalsm — APPROVED** [review]
Thanks again @noahliechti 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @noahliechti for docs

**Comment by noahliechti** [comment]
> Thanks again @noahliechti
> 
> Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

You are welcome! I'd love to contribute more!

**Pull request merged**